internal/server/commonpool.go

```diff
@@ -3,61 +3,29 @@ import (
 	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 )
 
-
 //islandDistribute distributes resources to island as requested 
 func (s *SOMASServer) islandDistribute(resource int, islandID shared.ClientID) { 
-
-	resChan := make(chan clientInfoUpdateResult, 1)
-
 	ci := s.gameState.ClientInfos[islandID]
-	ci.Resources += resource 	
-	resChan <- clientInfoUpdateResult{
-		ID:  islandID,
-		Ci:  ci,
-		Err: nil,
-	}			
+	ci.Resources = ci.Resources + resource 				
```

Comment:
This should be logged in the console saying that the client has gained X resources and the common pool has lost X resources. 
Also potentially there should be a function in the client notifying them that they have received resources from the common pool so that they can update their internal state.